src/pages/Shop/Components/ShopFilter/ShopFilter.js

```diff
@@ -0,0 +1,32 @@
+import React, {Component} from 'react';
+
+import './ShopFilter.scss'
+class ShopFilter extends Component {
+  render() {
+    const {selectFilter} = this.props
+    return(
+        <select onChange={selectFilter} className="ShopFilter" name="ordering">
+        {
+          Object.entries(FILTER_MENU).map((menu, idx) => {
+            return (
+              <option key={idx+1} value={menu[1]}>
+                {menu[0]}
+              </option>
+            )
+          })
```

Comment:
```suggestion
          Object.entries(FILTER_MENU).map((menu, idx) => {
            const [text, value] = menu
            return (
              <option key={idx+1} value={value}>
                {text}
              </option>
            )
          })
```
혹은
```suggestion
          Object.entries(FILTER_MENU).map(([text, value], idx) => {
            return (
              <option key={idx+1} value={value}>
                {text}
              </option>
            )
          })
```
이러한 방식으로 구조분해 할당을 사용해주시면 해당 변수가 하는 일을 더욱 잘 사용하는 코드를 작성할 수 있습니다.

Reply:
넹 반영하였습니당 👍